src/shared/model/Routes.ts

```diff
@@ -10,13 +10,12 @@ export type RoutePaths =
   // router between these comments are for legacy reasons and could be refactored
   | '/reset'
   | '/reset/email-sent'
-  | '/reset/complete'
+  // end routes to be refactored
   | '/reset-password'
+  | '/reset-password/expired'
   | '/reset-password/:token'
-  | '/password/reset-confirmation'
-  | '/reset/resend'
-  | '/reset/expired'
-  // end routes to be refactored
+  | '/reset-password/complete'
+  | '/reset-password/resend'
```

Comment:
I wonder if it's worth refactoring `/reset` into `/reset-password` like you did with some of these so that they're all the same and clear what they're doing.

And then add a rewrite rule in fastly for backwards compatibility?

```vcl
if (req.url ~ "^/reset/") {
    set req.url = regsub(req.url, "^/reset/","/reset-password/");
}
```